flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/TaskIOMetricGroup.java

```diff
@@ -178,6 +204,17 @@ private double getBusyTimePerSecond() {
         return busyTimeEnabled ? 1000.0 - Math.min(busyTime, 1000.0) : Double.NaN;
     }
 
+    private double getAccumulatedBusyTime() {
+        return busyTimeEnabled
+                ? Math.max(
+                        System.currentTimeMillis()
+                                - taskStartTime
```

Comment:
Minor: I'd suggest to add an assertion (`checkState(this.taskStartTime != 0)`) to make sure `markTaskStart()` has been called.